src/lavinmq/http/controller/queues.cr

```diff
@@ -114,6 +114,15 @@ module LavinMQ
           end
         end
 
+        put "/api/queues/:vhost/:name/restart" do |context, params|
+          with_vhost(context, params) do |vhost|
+            refuse_unless_management(context, user(context), vhost)
+            q = find_queue(context, params, vhost)
+            q.restart!
```

Comment:
`restart!` is a no-up, just returns when the queue is running ( `unless @closed`) 

Maybe `recover` would be a better name? 

And also, have a response that say that the queue was restarted/recovered so you dont issue a PUT to this endpoint and you get nothing back but also nothing happens with the queue.